src/app/modules/landing/modules/auth/components/login/login.component.ts

```diff
@@ -43,10 +43,10 @@ export class LoginComponent {
         },
         (err: HttpErrorResponse) => {
           this.store.dispatch(new ToasterError(extractError(err)));
-          event.target.disabled = false;
+          this.submitClicked = false;
         },
```

Comment:
you have to subscribe to AuthLogin action

Reply:
Ok but i am ready in the subscribe of the dispatch of AuthLogin action, what do you mean ?